### ISSUE TITLE
finishing synthesis no longer auto-starts the next synthesis (vanilla client)

### ISSUE THREAD

**Issue by armando1793** [hikarin-dev/Auto-Crafter#2]
Recently observed and I'm not sure how to fix

Previously, the trigger would auto-start the next synthesis step unless stop conditions are met. At the moment this is not working

I have reviewed README.md for any instruction updates, so far everything seems to have been setup correctly. I also manually tested NUM0 to see if pressing navigates to the crafting stage and it does

For context: macro being triggered is multi-part. Requiring 2 parts before finishing the item and restarting

Anything I should do to fix on my client? Can provide additional info if necessary although I'm not sure what I should provide

Thanks

**Comment by hikarin-dev** [comment]
There should be an overlay at the top right of your screen about the status of the auto-crafter.
![image](https://user-images.githubusercontent.com/19721540/226571000-c04ae261-722d-496e-ac73-46e6c5015a10.png)

On top of my head, here are the condition required for the next synthesis to proceed that you can check:
1. Craft limit is set to a number that is equal or lower than the current synthesized count. [MAT:#/#]
2. Spiritbond tracker should be less than 12. [SB:#], to reset this number, extract your Materias or restart ACT
3. The log line "You synthesize a ..." is not getting logged properly in your log files or not matching what the trigger is expecting. To check, open your most recent log files located at `%appdata%\Advanced Combat Tracker\FFXIVLogs`, find a log line similar to this `00|2023-03-21T17:47:27.0000000+08:00|08C2||You synthesize a circle of leather .|fc71b368117bb3c6`, paste it in the given url below to confirm the regular expression matches your log line.
https://regex101.com/r/p8piHh/1
4. The current version of my ACT parsing plugin is `2.6.8.1`, and my Triggernometry is at version `1.1.6.9`. I cannot confirm at the moment if another version broke this trigger, but testing it in vanilla client with those act plugin version leads me to no issues.

**Comment by armando1793** [comment]
Thank you for your response

I think it was because of #2. Prior to this I inexpicably had the plugin stop the crafting process at 6/12 max spiritbond items so I extracted and this was when the issue started. I just noticed had accidentally enabled both the remote plugin and the local plugin for some reason and disabling one may have messed up the tracking number of the other. An ACT reset fixed it. I checked items 1, 3, and 4 and they were not the issue

Again thank you so much for your time.
